lib/maplibre-compose-material3/src/iosMain/kotlin/dev/sargunv/maplibrecompose/material3/util.ios.kt

```diff
@@ -0,0 +1,24 @@
+package dev.sargunv.maplibrecompose.material3
+
+import dev.sargunv.maplibrecompose.material3.controls.ScaleBarMeasure
+import platform.Foundation.NSLocale
+import platform.Foundation.countryCode
+import platform.Foundation.currentLocale
+import platform.Foundation.languageCode
+
+internal actual fun defaultLanguage(): String? =
+  NSLocale.currentLocale.languageCode
+
+internal actual fun defaultCountry(): String? =
+  NSLocale.currentLocale.countryCode
+
+internal actual fun scaleBareMeasurePreference(): ScaleBarMeasure? =
+  null
+  /* TODO iOS/Mac developer:
+  when (NSLocale.currentLocale.measurementSystem) {
+    Locale.MeasurementSystem.metric -> ScaleBarMeasure.Metric
+    Locale.MeasurementSystem.uk -> ScaleBarMeasure.ImperialAndMetric
+    Locale.MeasurementSystem.us -> ScaleBarMeasure.Imperial
+    else -> ScaleBarMeasure.ImperialAndMetric
+  }
```

Comment:
@sargunv `measurementSystem` is only available from iOS 16, not sure how to only access it when on iOS 16 and above. Also, not sure about the syntax, as I don't get any help from the IDE on a non-Mac-system.